DataStructures/SortedList/SortedList.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using Utilities.Exceptions;
+
+namespace DataStructures.SortedList
+{
+    /// <summary>
+    /// Implementation of a Sorted List. It uses Quick Sort Algorithm.
+    /// </summary>
+    /// <typeparam name="T">Generic Type.</typeparam>
+    public class SortedList<T> : IComparisonSorter<T>, IEnumerable<T>
```

Comment:
```suggestion
    public class SortedList<T> : IEnumerable<T>
```
This is not a sorter